toolkit/scripts/toolchain/cross/gcc_cross.sh

```diff
@@ -0,0 +1,86 @@
+# Docs used to create this script:
+# https://preshing.com/20141119/how-to-build-a-gcc-cross-compiler/
+# https://mdeva2.home.blog/2019/07/08/building-gcc-as-a-cross-compiler-for-raspberry-pi/
+# http://jiadongsun.cc/2019/09/03/Cross_Compile_Gcc/#6-build-glibc
+
+installDir="/opt/cross"
+buildDir="$HOME/cross"
+scriptDir="$( cd "$( dirname "$0" )" && pwd )"
+
+sudo rm -rf ${installDir}
+sudo rm -rf ${buildDir}
+
+mkdir ${buildDir}
+cd ${buildDir}
+
+# Download source tarballs
+wget http://ftp.gnu.org/gnu/binutils/binutils-2.32.tar.xz
+wget https://ftp.gnu.org/gnu/gcc/gcc-9.1.0/gcc-9.1.0.tar.xz
+wget https://github.com/microsoft/WSL2-Linux-Kernel/archive/linux-msft-5.4.83.tar.gz
+wget https://ftp.gnu.org/gnu/glibc/glibc-2.28.tar.xz
+wget https://ftp.gnu.org/gnu/mpfr/mpfr-4.0.1.tar.gz
+wget http://ftp.gnu.org/gnu/gmp/gmp-6.1.2.tar.xz
+wget https://ftp.gnu.org/gnu/mpc/mpc-1.1.0.tar.gz
+
+# Install dependencies
+sudo apt-get install -y gawk rsync bison
```

Comment:
I believe make, patch, texinfo and diffutils are also required after testing in a Mariner image.

Reply:
I added a comment to install prerequisites instead of installing them in the script as per our offline discussion